### PR TITLE
 fix(installer): Phinx upgrade 

### LINKED ISSUE #12805: Getting issue while install of Elgg 3.1.2
Hi,

i am getting issue while install of Elgg 3.1.2 on ``Database installation`` step ie. 3rd

img_20190829

cant proceed to next step
installation doesn't shown any error message just this message mentioned in image

### PR DESCRIPTION
**Pull request by RiverVanRain**

### CODE REVIEW

**Comment by RiverVanRain** [comment]
These issues are solved with Phinx v0.11.1

**Comment by jeabakker** [comment]
you also need to update the `composer.lock`

File: composer.lock [hunk 1]
```diff
@@ -2424,7 +2424,7 @@
         },
         {
             "name": "robmorgan/phinx",
-            "version": "0.10.8",
+            "version": "0.11.1",
```
Comment (jeabakker) [inline composer.lock]: this is not the way to update a `composer.lock` file. You need to run the command `composer update robmorgan/phinx` to only update the phinx dependency

**Comment by jeabakker** [comment]
replaced by #12815

**Pull request closed without merging**